Tests/DatadogTests/Datadog/WebView/WKUserContentController+DatadogTests.swift

```diff
@@ -180,7 +180,7 @@ class WKUserContentController_DatadogTests: XCTestCase {
         )
 
         let messageHandler = try XCTUnwrap(controller.messageHandlers.first?.handler) as? DatadogMessageHandler
-        let webLogMessage = MockScriptMessage(body: #"{"eventType":"log","event":{"date":1635932927012,"error":{"origin":"console"},"message":"console error: error","session_id":"0110cab4-7471-480e-aa4e-7ce039ced355","status":"error","view":{"referrer":"","url":"https://datadoghq.dev/browser-sdk-test-playground"}},"tags":["browser_sdk_version:3.6.13"]}"#)
+        let webLogMessage = MockScriptMessage(body: #"{"eventType":"log","event":{"date":1635932927012,"error":{"origin":"console"},"message":"console error: error","session.id":"0110cab4-7471-480e-aa4e-7ce039ced355","status":"error","view":{"referrer":"","url":"https://datadoghq.dev/browser-sdk-test-playground"}},"tags":["browser_sdk_version:3.6.13"]}"#)
```

Comment:
Was this mock message made to be accurate to what the Browser SDK sends us? If so we probably shouldn't change it and verify we still send the proper attributes.

Reply:
You're right, my initial approach doesn't address logs coming from Browser SDK correctly.
Thanks for raising it.